apps/client/src/pages/login/page/login.tsx

```diff
@@ -0,0 +1,102 @@
+import { Header } from '@confeti/design-system';
+import {
+  BtnDeleteBlack20,
+  IcKakao,
+  IcApple,
+  ImgTypelogoBig,
+  Confeti3DLogo21,
+} from '@confeti/design-system/icons';
+import { Button } from '@confeti/design-system';
+import * as styles from './login.css';
+import { routePath } from '@shared/constants/path';
+
+const DESCRIPTION_TEXT =
+  '가입 시, confeti의\n[이용약관] 및 [개인정보처리방침]에 동의하게 돼요.';
+const LINK_MAP: LinkMap = {
+  이용약관: routePath.PRIVACY_CONFETI,
+  개인정보처리방침: routePath.PRIVACY_PERSONAL,
+};
+
+interface LinkMap {
+  [key: string]: string;
+}
+
+const parseLinkContent = (
+  part: string,
+  lineIndex: number,
+  index: number,
+): JSX.Element | string => {
+  const bracketMatch = part.match(/\[(.*?)\]/);
+  if (bracketMatch) {
+    const content = bracketMatch[1];
+    const link = LINK_MAP[content];
+
+    if (link) {
+      return (
+        <a
+          key={`${lineIndex}-${index}-${content}`}
+          href={link}
+          className={styles.atagText}
+          target="_blank"
+          rel="noreferrer"
+        >
+          {content}
+        </a>
+      );
+    }
+  }
+  return part;
+};
+
+const processLine = (line: string, lineIndex: number): JSX.Element => {
+  const parts = line.split(/(\[.*?\])/);
+  const processedParts = parts.map((part, index) =>
+    parseLinkContent(part, lineIndex, index),
+  );
+
+  return <p key={`line-${lineIndex}`}>{processedParts}</p>;
+};
+
+const parseDescription = (text: string): (JSX.Element | string)[] | null => {
+  if (!text) return null;
```

Comment:
```suggestion
const parseDescription = (text: string): (JSX.Element | string)[] => {
  if (!text) return [];
```
null 대신 [] 반환시키는 건 어떨까요? 빈 배열로 반환하면 타입이 항상 일정하게 유지돼서 좋을 거 같은데,,,
기존에는 `(JSX.Element | string)[] | null` 타입이었는데 null을 없애면 `(JSX.Element | string)[]` 타입으로 일관성을 유지할 수 있지 않을까요?